docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -127,26 +127,26 @@ During the app setup, if the Check Connection Options URL is configured for the
 
 <!-- theme:  -->
 
-## Validation Credentials
+## Validation credentials
 > The step of validating the credentials is optional. It does not change how the app operates. It is best practice to authenticate the user against your database or the downstream provider service.
 
 </div>
 </div>
 </div>
 
-Once the app is installed, it will be made available for configuration by merchants and API users. A merchant can navigate to the Shipping Manager and enable, configure and disable the carrier for any defined zone.
+Once you install the app, it will be made available for configuration by merchants and API users. A merchant can navigate to the Shipping Manager and enable, configure, and disable the carrier for any defined zone.
 
-## API Installation Workflow
+## API installation workflow
 
-To set up a carrier using the API, first connect it using the Connect Carrier API. Make a request containing the connection settings required by your carrier. The ID of the carrier is required. The carrier ID will be issued by BigCommerce when your carrier is registered. All connection fields are unique per carrier. If your carrier doesn’t require any connection settings then this object can be left empty.
+To set up a carrier using the API, first, connect it using the Connect Carrier API. Make a request containing the connection settings required by your carrier. The ID of the carrier is required. The carrier ID will be issued by BigCommerce when your carrier is registered. All connection fields are unique per carrier. If your carrier doesn’t require any connection settings, then this object can be left empty.
 
 <!--
 title: "Sample Request "
 subtitle: "POST https://developerserver.com/shipping/carrier/connection"
 lineNumbers: true
 -->
 
-**Example Request Carrier Connection**
+**Example request carrier connection**
 `/POST https://developerserver.com/shipping/carrier/connection`

```

Comment:
I think we can remove this line of code or add a link to the [Create a Carrier Connection](https://developer.bigcommerce.com/api-reference/store-management/shipping-api/shipping-carrier/postshippingcarrierconnection) endpoint.

Reply:
I will remove the code.